### PR TITLE
196190 change image error

### PR DESCRIPTION
**Pull request by adunndevster**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/5733)

### CODE REVIEW

**Comment by yufeih** [comment]
@adunndevster Test failed: `ContentImageTestBlock_InvalidImage_MissingAlt `

File: src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonParser.cs [hunk 1]
```diff
@@ -39,8 +39,8 @@ public override BlockState TryOpen(BlockProcessor processor)
 
             var extensionName = "triple-colon";
             Action<string> logError = (string message) => _context.LogError(
-                $"invalid-{extensionName}",
```
Comment (herohua) [inline src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonParser.cs:42]: + @superyyrrzz @fenxu be aware of this change

Comment (adunndevster) [reply src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonParser.cs:42]: @herohua - I was actually wondering about changing this pattern... Is this okay for me to do?

Comment (fenxu) [reply src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonParser.cs:42]: Docs build side should make some adjustment if you change this pattern, not for free.

Comment (adunndevster) [reply src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonParser.cs:42]: Okay, that makes sense. I will just undo this pattern for now then.

File: src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ImageExtension.cs [hunk 2]
```diff
@@ -110,7 +109,11 @@ public bool TryProcessAttributes(IDictionary<string, string> attributes, out Htm
 
                 if (!string.IsNullOrEmpty(currentLightbox))
                 {
-                    renderer.WriteLine($"<a href=\"{currentLightbox}#lightbox\" data-linktype=\"relative-path\">");
+                    var lighboxHtmlAttributes = new HtmlAttributes();
+                    lighboxHtmlAttributes.AddProperty("href", $"{currentLightbox}#lightbox");
+                    lighboxHtmlAttributes.AddProperty("data-linktype", $"relative-path");
+                    //renderer.WriteLine($"<a href=\"{currentLightbox}#lightbox\" data-linktype=\"relative-path\">");
```
Comment (jamarw) [inline src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ImageExtension.cs:115]: Hi @adunndevster , should this commented code be removed or kept as a reference?  Thanks

Comment (adunndevster) [reply src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ImageExtension.cs:115]: Oh, lemme remove!

**Pull request merged**